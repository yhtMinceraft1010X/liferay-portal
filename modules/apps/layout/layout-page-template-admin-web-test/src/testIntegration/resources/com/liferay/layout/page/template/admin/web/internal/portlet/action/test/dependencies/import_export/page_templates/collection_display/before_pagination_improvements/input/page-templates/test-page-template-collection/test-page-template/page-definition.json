{
	"pageElement": {
		"pageElements": [
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 100,
					"numberOfItemsPerPage": 10,
					"paginationType": "Numeric",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 100,
					"numberOfItemsPerPage": 10,
					"paginationType": "Simple",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 100,
					"numberOfItemsPerPage": 5,
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 5,
					"numberOfItemsPerPage": 22,
					"paginationType": "Numeric",
					"showAllItems": true,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 200,
					"numberOfItemsPerPage": 5,
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"numberOfColumns": 1,
					"numberOfItems": 5,
					"numberOfItemsPerPage": 5,
					"showAllItems": false
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 5,
					"numberOfItemsPerPage": 5,
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			}
		],
		"type": "Root"
	},
	"settings": {
		"colorSchemeName": "01",
		"themeName": "Classic"
	},
	"version": 1.1
}