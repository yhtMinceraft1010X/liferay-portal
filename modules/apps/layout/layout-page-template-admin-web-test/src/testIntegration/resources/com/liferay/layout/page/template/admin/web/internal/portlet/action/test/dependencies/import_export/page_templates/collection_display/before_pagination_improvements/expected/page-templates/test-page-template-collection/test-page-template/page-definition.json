{
	"pageElement": {
		"pageElements": [
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": false,
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 100,
					"numberOfItemsPerPage": 10,
					"numberOfPages": 10,
					"paginationType": "Numeric",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": false,
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 100,
					"numberOfItemsPerPage": 10,
					"numberOfPages": 10,
					"paginationType": "Simple",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": false,
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 100,
					"numberOfItemsPerPage": 5,
					"numberOfPages": 20,
					"paginationType": "None",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": true,
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 5,
					"numberOfItemsPerPage": 22,
					"numberOfPages": 1,
					"paginationType": "Numeric",
					"showAllItems": true,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": false,
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 200,
					"numberOfItemsPerPage": 5,
					"numberOfPages": 40,
					"paginationType": "None",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": false,
					"numberOfColumns": 1,
					"numberOfItems": 5,
					"numberOfItemsPerPage": 5,
					"numberOfPages": 1,
					"paginationType": "None",
					"showAllItems": false
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			},
			{
				"definition": {
					"collectionConfig": {
						"collectionReference": {
							"className": "com.liferay.asset.list.model.AssetListEntry",
							"classPK": "${CLASS_PK}"
						},
						"collectionType": "Collection"
					},
					"collectionViewports": [
						{
							"collectionViewportDefinition": {
							},
							"id": "landscapeMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "portraitMobile"
						},
						{
							"collectionViewportDefinition": {
							},
							"id": "tablet"
						}
					],
					"displayAllItems": false,
					"displayAllPages": false,
					"listItemStyle": "",
					"listStyle": "",
					"numberOfColumns": 1,
					"numberOfItems": 5,
					"numberOfItemsPerPage": 5,
					"numberOfPages": 1,
					"paginationType": "None",
					"showAllItems": false,
					"templateKey": ""
				},
				"pageElements": [
					{
						"definition": {
							"collectionItemConfig": {
							}
						},
						"type": "CollectionItem"
					}
				],
				"type": "Collection"
			}
		],
		"type": "Root"
	},
	"settings": {
		"colorSchemeName": "01",
		"themeName": "Classic"
	},
	"version": 1.1
}